documentation/asciidoc/topics/proc_adding_labels_annotations_env.adoc

```diff
@@ -12,12 +12,12 @@ Add labels and annotations to your {ispn_operator} subscription with the `spec.c
 +
 [source,options="nowrap",subs=attributes+]
 ----
-{oc_edit} subscription {subscription_name} -n openshift-operators
+{oc_edit} subscription {subscription_name} -n {example_namespace}
```

Comment:
Atm {example_namespace} resolves in the community to ispn-namespace and downstream to rhdg-namespace but on the openshift console I see  openshift-operators.
Does it make sense to use openshift-operators as a example namespace in the downstream docs?

Reply:
Downstream we should be using `openshift-operators` and upstream just `operators` when describing interactions with the Operator Subscription.